source/extensions/common/wasm/context.cc

```diff
@@ -1468,12 +1468,22 @@ WasmResult Context::sendLocalResponse(uint32_t response_code, absl::string_view
                                       Pairs additional_headers, uint32_t grpc_status,
                                       absl::string_view details) {
 
-  auto modify_headers = [additional_headers](Http::HeaderMap& headers) {
-    for (auto& p : additional_headers) {
-      const Http::LowerCaseString lower_key{std::string(p.first)};
-      headers.addCopy(lower_key, std::string(p.second));
+
```

Comment:
Nit: 2 empty lines, which breaks the format check.

Reply:
Weird -- I must have the wrong version of clang-format or something because it passes for me. But it's easy to fix.

Reply:
Thanks!